bindings/java/src/org/sleuthkit/datamodel/BlackboardArtifact.java

```diff
@@ -193,6 +195,15 @@ public String getArtifactTypeName() {
 		return this.artifactTypeName;
 	}
 
+	/**
+	 * Gets the artifact type category for this artifact.
+	 *
+	 * @return The artifact type category.
+	 */
+	public CategoryType getCategoryType() {
+		return ARTIFACT_TYPE.fromID(artifactTypeId).getCategory();
+	}
```

Comment:
I don't think this method is needed at this time. Please remove it until we have a use case for it, at which point it would probably be better to just return the entire Type object and let the caller query the Type for its Category.